### PR TITLE
[OSJC-275] Add volumes to deploymentconfigs

### PR DESCRIPTION
**Pull request by jcantrill**
This PR provides convenience methods for adding volumes to the DC podspec

### CODE REVIEW

**Comment by jcantrill** [comment]
cc @adietish  [test]

File: src/main/java/com/openshift/internal/restclient/model/ReplicationController.java [hunk 1]
```diff
@@ -365,8 +365,21 @@ public void addVolume(IVolumeSource volumeSource) {
...
+	public <T extends IVolumeSource> T addVolume(String volumetype, String name) {
+    	ModelNode volList = get(VOLUMES);
+    	ModelNode node = volList.add();
+    	node.get(volumetype).set(new ModelNode());
+    	IVolumeSource source = VolumeSource.create(node);
+    	source.setName(name);
+		return (T) source;
+	}
+
+	@Override
```
Comment (adietish) [inline src/main/java/com/openshift/internal/restclient/model/ReplicationController.java]: dare to correct identation for great and consistent code formatting experience?

**Comment by jcantrill** [comment]
[merge]

File: src/main/java/com/openshift/internal/restclient/model/ReplicationController.java (hunk 1 above)
Comment (jcantrill) [inline src/main/java/com/openshift/internal/restclient/model/ReplicationController.java]: fixed.  Modified my IDE to replace all tabs with spaces

**Pull request merged**